internal/exec/stages/files/files.go

```diff
@@ -30,6 +32,9 @@ import (
 
 const (
 	name = "files"
+
+	// see https://github.com/systemd/systemd/pull/11131/files#diff-9e1684efe63d10844b8dd6708fc60e8dR414
```

Comment:
Minor: I'd just link to the merged commit itself: https://github.com/systemd/systemd/commit/65e183d7899eb3725d3009196ac4decf1090b580